pkg/lib/cockpit-components-listing-panel.jsx

```diff
@@ -73,19 +71,17 @@ export class ListingPanel extends React.Component {
                 delete loadedTabs[prevTab];
 
             // ensure the new tab is loaded and update state
-            loadedTabs[tabIdx] = true;
-            this.setState({ loadedTabs: loadedTabs, activeTab: tabIdx });
+            loadedTabs[result.itemId] = true;
+            this.setState({ loadedTabs: loadedTabs, activeTab: result.itemId });
         }
-        e.stopPropagation();
-        e.preventDefault();
     }
 
     render() {
         const links = this.props.tabRenderers.map((itm, idx) => {
             return (
-                <li key={idx} className={ (idx === this.state.activeTab) ? "active" : ""}>
-                    <a href="#" tabIndex="0" onClick={ this.handleTabClick.bind(self, idx) }>{itm.name}</a>
-                </li>
+                <NavItem key={idx} id={itm.id} itemId={idx} isActive={idx === this.state.activeTab}>
+                    {itm.name}
+                </NavItem>
```

Comment:
This is a11y regression. Now the navigation is not reachable by keyboard but it used to be.
I cannot make sense of this, as this NavItem is rendered as `div` but according to documentation it should be `a` by default (which would be accessible). Maybe it is worth reporting (or diving deep how it actually should work?)

We have similar pattern in selinux and it works there as expected.